lib/semantic_puppet.rb

```diff
@@ -0,0 +1,9 @@
+require 'gettext-setup'
+
+module SemanticPuppet
+  GettextSetup.initialize(File.absolute_path('../locales', File.dirname(__FILE__)))
```

Comment:
Is the plan still to avoid a hard dependency on `gettext-setup`? If so is that change intended for a separate PR?